content/kb/troubleshooting-recurring-job-stuck-after-detach-attach.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Troubleshooting: Recurring job does not create new jobs after detach and attach"
+author: Chin-Ya Huang
+draft: false
+date: 2021-04-21
+categories:
+  - "recurring job"
+---
+
+## Applicable versions
+
+All Longhorn versions.
```

Comment:
Should we indicate `Version v1.1.1 and earlier` ? So, we fix this in v1.1.2 we don't have to modify the KB

Reply:
I have no problem having the version pre-set before the fix, but thought it might make more sense to be in sync with the current status. @innobead , can we get some opinion on this?

Reply:
I think we can revisit back later when we really fix the issue. @PhanLe1010, update this action item in the related ticket? Thanks.